Clinical-Genomics/cgbeacon2
# Fix podman systemd docs

### This PR adds | fixes:
- Fix #192: documentation on how to deploy the beacon as a service (podman systemd) was no longer valid

### How to test:
- Follow the instructions on a linux VM and make sure the described containers can be create and work as expecyed

### Review:

This [version](https://semver.org/) is a:
